almalinux-deploy.sh

```diff
@@ -495,11 +495,21 @@ distro_sync() {
     fi
     local -r step='Run dnf distro-sync -y'
     local ret_code=0
-    dnf distro-sync -y || {
+    local dnf_repos="--enablerepo=powertools"
+    # create needed repo
+    if [ "${panel_type}" == "plesk" ]; then
+        plesk installer --select-release-current --show-components --skip-cleanup
+        dnf_repos+=",PLESK_*-dist"
+    fi
+    dnf distro-sync -y "${dnf_repos}" || {
         ret_code=${?}
         report_step_error "${step}. Exit code: ${ret_code}"
         exit ${ret_code}
     }
+    # remove unnecessary repo
+    if [ "${panel_type}" == "plesk" ]; then
+        plesk installer --select-release-current --show-components
```

Comment:
This code will be unreachable if distro-sync failed

Reply:
Yes. And what?